addons/l10n_tr/__manifest__.py

```diff
@@ -2,18 +2,20 @@
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
 {
-    'name': 'Turkey - Accounting',
+    'name': 'Türkiye - Accounting',
```

Comment:
Don't translate the names (also in data files). These names are in English: "Turkey"
If you want translations do them in the po file.

Reply:
This not a translation, this is the new name of the Turkish republic https://turkiye.un.org/en/184798-turkeys-name-changed-t%C3%BCrkiye